README.md

```diff
@@ -161,49 +161,49 @@ FLAGS:
 
 
 OPTIONS:
-        --auction-data-page-size <auction-data-page-size>
```

Comment:
These should be updated in `main.rs` as well. The `#[structopt(...)]` attributes need to include a `name = 'some name'` since the goal here was to keep this auto-generated.

Reply:
You can then run `cargo run --package driver -- --help` to get the full help text.

Reply:
Right thanks. This may wind up being less trivial than anticipated.